lib/model/query/audits.js

```diff
@@ -31,15 +33,15 @@ values (${actorId}, ${action}, ${acteeId}, ${(details == null) ? null : JSON.str
 // TODO: some sort of central table for all these things, not here.
 const actionCondition = (action) => {
   if (action === 'nonverbose')
-    return sql`action not in ('submission.create', 'submission.attachment.update', 'backup')`;
+    return sql`action not in ('submission.create', 'submission.update', 'submission.update.version', 'submission.attachment.update', 'backup')`;
   else if (action === 'user')
     return sql`action in ('user.create', 'user.update', 'user.delete', 'assignment.create', 'assignment.delete')`;
   else if (action === 'project')
     return sql`action in ('project.create', 'project.update', 'project.delete')`;
   else if (action === 'form')
     return sql`action in ('form.create', 'form.update', 'form.delete', 'form.attachment.update', 'form.update.draft.set', 'form.update.draft.delete', 'form.update.publish')`;
   else if (action === 'submission')
-    return sql`action in ('submission.create', 'submission.attachment.update')`;
+    return sql`action in ('submission.create', 'submission.update', 'submission.update.version', 'submission.attachment.update')`;
 
   return sql`action=${action}`;
 };
```

Comment:
Related to #274, could we also add `if`s for `action === 'field_key'` and `action === 'public_link'`?

Also, I'm noticing that the action filter above the audit log will filter if `session.end` (Delete) is selected, but not if `session` (Session Actions) is selected. What do you think about adding `action === 'session'` as well?

https://user-images.githubusercontent.com/5970131/113957871-14171600-97ee-11eb-9434-7a4a2e52d7b0.mov

Reply:
I just noticed one other thing, which is that an `assignment.create` action is logged for a public link when the public link is created. That makes sense to me; however, right now Frontend will label the action "User > Assign Role" even though the actee isn't a user.

I think the easiest way to account for this case would be for Frontend to label `assignment.create` differently: something like "Role > Assign" or "Assignment > Create". (`assignment.delete` would become something like "Role > Revoke".) One thing I like about this approach is that it mirrors how Backend organizes these actions, as being about assignments rather than strictly users.

If `assignment.create` and `assignment.delete` aren't just about users, then I think they can be removed from the `action === 'user'` case above. Then we'd have a situation where the actions in each list have the same prefix: filtering on `'project'` filters for a list of actions that all begin with `project.`, filtering on `'user'` filters for a list of actions that all begin with `user.`, and so on. Maybe that would simplify `actionCondition()`?

Reply:
i made a new commit. does it look good to you?

Reply:
Yes, looks great! I think it'd be great to add `field_key.delete` and `public_link.delete` as well, but I'll mention that in a separate comment below.

Reply:
I've tried to summarize the audits-related changes in the release criteria. Let me know if I missed anything! I've also specified new text for Frontend there, so let me know also if anything there should change.